CONTRIBUTING.md

```diff
@@ -56,6 +56,35 @@ This project follows the
 [Oracle Java code conventions](http://www.oracle.com/technetwork/java/codeconvtoc-136057.html)
 and the Scala conventions detailed above, preferring the latter.
 
+### Test
+There are two types of tests in this project. Unit tests that are written in Scala, and integration tests that are written in python. 
+We encourage writing integration tests if you have to choose between the two as that helps us ensure the plugin is working across different platforms. 
+
+#### Unit tests
+Unit-tests are located in com.nvidia:rapids-4-spark-tests_2.12. In order to run the unit-tests follow these steps
+1. Build the parent project. That will build the entire project including tests
```

Comment:
Similarly I think parent project is more nebulous than it needs to be here.  "Build from the top of the repository" would be clearer, IMHO.